posts/2022-01-20-ssl-certificates.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+blog_description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+
+== All Beta Features package
+The All Beta features package for link:{url-about}[Open Liberty] contains all Open Liberty beta features and GA features and functions.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try the beta package, <<feedback, let us know what you think>>.
+
+The following beta features are available in 22.0.0.2-beta:
+
+* <<ssl, Support new change in the SSL certificate with urlencoded string>>
+
+[#ssl]
+=== Support new change in the SSL certificate with urlencoded string
+
+An X.509 certificate is a digital certificate that is used to manage identity and security in internet communications and computer networking.
+
+Intermediate servers, such as a web server, proxy server, or load balancer can send X.509 certificates to Open Liberty servers. Due to the deprecation of long lines folding in the `HTTP` header continuation, some open source intermediate servers might send a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded format.
+
+Prior to Open Liberty 22.0.0.2-beta, this new url-encoded certificate was not supported. Any request with this new format was rejected and the request terminated.
```

Comment:
sorry- should've caught this before- I think line 40 and line 60 should swap places. The important info that we want to highlight is that the beta adds this capability. As is, that's not explained until the end of the section. Better to highlight it here and then mention previous behavior at the end.